app/templates/mock-login.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="au-c-main-container" itemprop="mainContentOfPage">
+    <div class="au-c-main-container__content au-c-main-container__content--scroll">
+        <main id="content">
+            <div class="au-o-region-large">
+                <div class="au-o-layout">
+                    <AuHeading @skin="4">{{t 'mockLogin.pick'}}</AuHeading>
+                    <MockLogin as |login|>
+                        {{#if this.queryStore.isRunning}}
+                        <AuLoader @size="small" />
+                        {{else}}
+                        {{#if login.errorMessage}}
+                        <AuAlert @alertIcon="alert-triangle" @alertTitle={{login.errorMessage}} @alertskin="warning" />
+                        {{/if}}
+                        <login.each-account @accounts={{this.model}} as |account|>
+                            <div class="au-o-box au-o-box--small au-c-card au-u-margin-bottom-small">
+                                <p>
+                                    {{account.user.fullName}}
+                                </p>
+                            </div>
+                        </login.each-account>
+                        <NumberPagination @page={{this.page}} @size={{this.size}} @nbOfItems={{this.model.length}}
```

Comment:
I would remove this. I doubt there will ever be so many roles / mock users that pagination is needed.